src/components/table/expandable/row/style.scss

```diff
@@ -8,7 +8,12 @@
 	box-shadow: 0px 1px 1px -1px rgb(var(--smoothly-table-border));
 }
 
-:host::slotted(:not(.detail)) {
+:host a { 
```

Comment:
A lot of this styling is applied to any `a` in the expandable row, not just the direct child.

Reply:
Same for a lot of the selectors here